element-templates/openbots_connector.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
   "name": "OpenBots Connector",
-  "id": "66cedae8-72e7-4fe3-b980-bf3862b28c0a",
-  "version": 1702470612996,
+  "id": "6a341cd2-8d96-4d03-bb3b-371b5b15fbcd",
```

Comment:
@ksikota I think also the ID should be the same as it was on the first version, otherwise if anyone has install it it might think it is a different connector.